library/src/scala/EnumValue.scala

```diff
@@ -0,0 +1,10 @@
+package scala
+
+trait EnumValue extends Product:
```

Comment:
For parity with case object this should extend Serializable too I think:
```suggestion
trait EnumValue extends Product with Serializable:
```

Reply:
Actually it probably makes more sense to have `Product` and `Serializable` added as parents to the enum class itself, that way they never show up when lubbing enum cases.

Reply:
Yes, but I am not sure about what other effects this will have.